crates/turborepo-lib/Cargo.toml

```diff
@@ -7,6 +7,7 @@ license = "MPL-2.0"
 [features]
 # Allows configuring a specific tls backend for reqwest.
 # See top level Cargo.toml for more details.
+default = ["rustls-tls"]
```

Comment:
Just verifying that this is meant to be part of this PR and didn't sneak in from dev workflow.

Reply:
I'll make this a separate PR but I got sick of getting the feature error message. If turbopack is cool with this, I'm just gonna turn it on here